src/Microsoft.AspNet.Razor/Chunks/ChunkTree.cs

```diff
@@ -3,11 +3,11 @@
 
 using System.Collections.Generic;
 
-namespace Microsoft.AspNet.Razor.Generator.Compiler
+namespace Microsoft.AspNet.Razor.Chunks
 {
-    public class CodeTree
+    public class ChunkTree
```

Comment:
Does this type provide anything besides acting as a level of indirection - i.e. could we have `ChunkTreeBuilder` return an `IReadOnlyList<Chunk>` rather than have this type?